File: wp_plugin/modules/cron.py
import wp_plugin.modules.plugin_module as m

class cron(m.plugin_module):

	def pre_process(self):
		super().pre_process()
		self.add_template('include/{{plugin_namespace}}/Cron/Cron.php')
		self.add_template('include/{{plugin_namespace}}/Cron/Job.php')

File: lib/components/cron.js
const BaseComponent = require('./base-component.js');

class CronComponent extends BaseComponent {

	async setup() {
		this.addTemplate( `include/{{plugin.namespace}}/Cron/Cron.php` )
		this.addTemplate( `include/{{plugin.namespace}}/Cron/Job.php` )
		super.setup();
	}

}

module.exports = CronComponent;
